src/components/Glp/GlpSwap.js

```diff
@@ -800,161 +800,176 @@ export default function GlpSwap(props) {
+                <div className="align-right fee-block">
+                  {isBuying && (
+                    <Tooltip
+                      handle={isBuying && isSwapTokenCapReached ? "NA" : feePercentageText}
+                      position="right-bottom"
+                      renderContent={() => {
+                        if (!feeBasisPoints) {
+                          return (
+                            <div className="text-white">
+                              <Trans>Fees will be shown once you have entered an amount in the order form.</Trans>
+                            </div>
+                          );
+                        }
                         return (
                           <div className="text-white">
-                            <Trans>Fees will be shown once you have entered an amount in the order form.</Trans>
+                            {feeBasisPoints > 50 && (
+                              <Trans>To reduce fees, select a different asset to pay with.</Trans>
+                            )}
+                            <Trans>Check the "Save on Fees" section below to get the lowest fee percentages.</Trans>
                           </div>
                         );
-                      }
-                      return (
-                        <div className="text-white">
-                          {feeBasisPoints > 50 && <Trans>To reduce fees, select a different asset to pay with.</Trans>}
-                          <Trans>Check the "Save on Fees" section below to get the lowest fee percentages.</Trans>
-                        </div>
-                      );
-                    }}
-                  />
-                )}
-                {!isBuying && (
-                  <Tooltip
-                    handle={feePercentageText}
-                    position="right-bottom"
-                    renderContent={() => {
-                      if (!feeBasisPoints) {
+                      }}
+                    />
+                  )}
+                  {!isBuying && (
+                    <Tooltip
+                      handle={feePercentageText}
+                      position="right-bottom"
+                      renderContent={() => {
+                        if (!feeBasisPoints) {
+                          return (
+                            <div className="text-white">
+                              <Trans>Fees will be shown once you have entered an amount in the order form.</Trans>
+                            </div>
+                          );
+                        }
                         return (
                           <div className="text-white">
-                            <Trans>Fees will be shown once you have entered an amount in the order form.</Trans>
+                            {feeBasisPoints > 50 && <Trans>To reduce fees, select a different asset to receive.</Trans>}
```

Comment:
Maybe create some flag variable like 
isFeesHigh = feeBasisPoints > FEES_HIGH_BPS

just there is a several places with check exact number `feeBasisPoints > 50`

Reply:
Makes sense, updated. Thanks  🙏